interface/forms/newpatient/templates/report.html.twig

```diff
@@ -0,0 +1,13 @@
+<div class="w-100">
+{% for e in encounters %}
+    {% set _reason = (e.reason) ? " - " ~ e.reason|nl2br|text : "" %}
```

Comment:
Note that the escaping with `text` needs to happen before the `nl2br` call or else the newlines will turn into `<br>`.